cmd/flags.go

```diff
@@ -163,6 +180,10 @@ func FlagProjectMetadataPath(projectMetadataPath *string) {
 	flagSet.StringVar(projectMetadataPath, "project-metadata", EnvOrDefault(EnvProjectMetadataPath, DefaultProjectMetadataPath), "path to project-metadata.toml")
 }
 
+func UpdateProjectMetadataPath(projectMetadataPath *string, platformAPI, layersDir string) {
```

Comment:
Instead of having a function that updates the value referenced by the pointer it might be more straightforward/readable to have a `DefaultProjectMetadataPath(platformAPI, layersDir string) string` function that simply returns the computed default. Then in places where we are currently doing things like 
```
cmd.UpdatePlanPath(&b.planPath, b.buildArgs.platformAPI, b.layersDir)
```
we could instead do something like
```
if b.planPath == cmd.PlaceholderPlanPath {
   b.planPath = cmd.DefaultPlanPath(b.platformAPI, b.layersDir)
}
```
In my opinion, the above usage makes it very clear to someone reading a given implementation of `Args` what exactly is happening and why, without a lot of digging.

Reply:
@yaelharel thoughts on this suggestion? I still think the suggested change would make the commands more readable.

Reply:
I'm really sorry, I missed this comment. I'll review it today.

Reply:
Your idea is great! I'll go and make this change.
Sorry again for missing your comment for so long...